### ISSUE TITLE
Unavoidable warning during `publish dev`

### ISSUE THREAD

**Issue by cconrad** [dendronhq/dendron#2450]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [x] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

When running `dendron publish dev`, I see the following warning:

```
WARN: --query and --vault parameter not implemented for podType export
```

## To Reproduce

1. Run `dendron publish dev`
4. See warning

## Expected behavior

The warning should not be shown, unless the user can do something to avoid it. That seems not to be the case here?

## Screenshots

![image](https://user-images.githubusercontent.com/584491/154710765-174a59ff-0943-431d-aacf-a291b437791d.png)

## Desktop (please complete the following information)

- OS: Windows
- Browser: Edge
- Version: 0.82.0 (CLI)

**Comment by SeriousBug** [comment]
Thanks for reporting this issue, I can see where it's coming from in the code but I'm not sure why the warning is in place.

@Harshita-mindfire could you take a look whenever you have time why the warning is displayed?

**Comment by Harshita-mindfire** [comment]
This should be fixed in v0.86 of Dendron. Feel free to re-open if not the case.
